packages/astro/src/vite-plugin-astro/index.ts

```diff
@@ -143,9 +144,15 @@ export default function astro({ config, logging }: AstroPluginOptions): vite.Plu
 					define: config.vite.define,
 				});
 
-				// Signal to Vite that we accept HMR updates
-				const SUFFIX = isProduction ? '' : `\nif (import.meta.hot) import.meta.hot.accept((mod) => mod);`;
-
+				let SUFFIX = '';
+				// Add HMR handling in dev mode.
+				if (!resolvedConfig.isProduction) {
```

Comment:
Why is this needed? Pretty sure we are already handling this elsewhere.

Reply:
Oh this, was already happening above, weird.

Reply:
I'm happy to remove but yea, would love to keep moving parts to a minimum in this PR